packages/kit/test/apps/basics/src/routes/endpoint-output/_tests.js

```diff
@@ -0,0 +1,26 @@
+import * as assert from 'uvu/assert';
+
+/** @type {import('test').TestMaker} */
+export default function (test) {
+	test('not ok on void endpoint', null, async ({ fetch }) => {
+		const res = await fetch('/endpoint-output/empty', { method: 'DELETE' });
+		assert.equal(res.ok, false);
+	});
+	test('200 status on empty endpoint', null, async ({ fetch }) => {
+		const res = await fetch('/endpoint-output/empty');
+		assert.equal(res.status, 200);
+		assert.equal(await res.json(), {});
```

Comment:
can we also test what the other return types would show for this?

Reply:
Do you mean other than `return {}` in the endpoints?

Reply:
I meant like `res.text()`, etc.

Reply:
I used `res.text()` in the other tests. I think we only need to check for `.json()` and `.text()` as they're the most commonly used ones. Other than that, just making sure that it doesn't throw on other methods are probably good enough.

Reply:
I think it'd be nice to have a test checking `/endpoint-output/empty` with `.text()`. I don't see that one in this file. Not sure if it lives elsewhere already

Reply:
That's true, but I don't think it's worth a separate test and `fetch` call. Response bodies can't be reused after it's been parsed and I think it's quite clear that `.text()` representation of `{}` would just be `'{}'`.

I'm just concerned that adding more `fetch` integration tests would make it even more flaky than it needs to be.